doc/classes/OptionButton.xml

```diff
@@ -165,6 +159,9 @@
 	<members>
 		<member name="action_mode" type="int" setter="set_action_mode" getter="get_action_mode" overrides="BaseButton" enum="BaseButton.ActionMode" default="0" />
 		<member name="alignment" type="int" setter="set_text_alignment" getter="get_text_alignment" overrides="Button" enum="HorizontalAlignment" default="0" />
+		<member name="item_count" type="int" setter="set_item_count" getter="get_item_count" default="0">
+			The number of items to select.
```

Comment:
```suggestion
			The number of items to select from.
```
Does it also include separators as the previous description mentioned? If so that should be kept.

Reply:
A separator is technically just another item, so mentioning it specifically isn't very useful.
![image](https://user-images.githubusercontent.com/2223172/146891912-be02aba1-daba-424b-a4cd-b0892fd29799.png)